bakerydemo/templates/base/basic_auth.html

```diff
@@ -0,0 +1,16 @@
+{% load wagtailadmin_tags %}
+
+<title>Authentication Required</title>
+<style>
+    h1 { font-size: 50px; }
+    body { font: 20px sans-serif; color: #333; }
+    article { display: block; text-align: center; margin: 0 auto; }
+    a { color: #dc8100; text-decoration: none; }
+    a:hover { color: #333; text-decoration: none; }
+</style>
+
+<article>
+    <h1>Authentication Required</h1>
+    <p>You must login to view this site</p>
+    <h3>Looking for Wagtail's demo instance? <a href="https://demo.wagtail.org">Click here</a>.</h3>
+</article>
```

Comment:
Even if this is just a holding page it still needs to be valid HTML and be accessible. Let me know if you want some help with this.

Specific things that stand out:

- Needs to be valid HTML
- Font sizes should always be in relative units
- Keep link underlines
- Never use "click here" as link text